src/styles/theme.ts

```diff
@@ -29,6 +29,16 @@ const theme = createTheme({
   },
   typography: {
     fontFamily: 'Montserrat',
+    h1: {
+      fontFamily: 'Mukta',
+      fontSize: '40px',
+      fontWeight: '600',
+    },
+    body1: {
+      fontFamily: 'Montserrat',
+      fontSize: '20px',
+      fontWeight: '600',
+    },
```

Comment:
I think these font sizes are way too large. Just take a look at the designs
<img width="745" alt="image" src="https://user-images.githubusercontent.com/1069628/159136786-b9f188ab-8714-4de7-93c9-043173748831.png">

vs how this PR looks on my screen
<img width="1569" alt="image" src="https://user-images.githubusercontent.com/1069628/159136809-aca372d7-d803-43ee-b65f-a0c7838fe194.png">

I think we should probably just leave this as it was before because it was close enough, and we can make these kind of adjustments later on.

Reply:
I used the Container component and put the entire section within that container. Here's an image of the same, is it fine? 
<img width="1013" alt="Screenshot 2022-03-20 at 3 28 14 PM" src="https://user-images.githubusercontent.com/86240862/159157142-3e5857f1-69d0-41b7-9d5b-f95465c383f7.png">

Reply:
Though the design is a bit different from the one on the Figma page. Earlier I tried to match values of the styles as mentioned on the Figma design page.Like margin of the paragraph from the image was 'x' px on the Figma design so I copied the same. But it looks different with this PR as you mentioned above.

Reply:
The section size is not like before, it is somewhat smaller in my opinion, while using the Container component.  I don't know if it is optimum . I put the header, paragraph and the image inside one single Container component. I tried using the maxWidth and sx props of the Container component but no success. Am I doing something wrong?

Reply:
<img width="1041" alt="Screenshot 2022-03-23 at 4 10 56 PM" src="https://user-images.githubusercontent.com/86240862/159710558-97c3ecf4-cf83-4078-8a1b-f8b4cf6ae606.png">

I wrapped this section in a Container component and added individual font sizes for the text using the sx prop(removed the fontSize from the theme.ts file). I think the size is small even though I used the maxWidth="xl". I also removed marginLeft from Box components and applied it to just the Container component. Can you check if it is fine now?